src/snackbar/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import I18nMixin from '@dojo/framework/widget-core/mixins/I18n';
+import Button from '../button/index';
+import * as css from '../theme/snackbar.m.css';
+import bundle from './nls/Snackbar';
+
+export interface SnackbarProperties {
+	success?: boolean;
+	open?: boolean;
+	title: string;
+	onDismiss?: () => void;
```

Comment:
are we planning on implementing a generic action for the snackbar too rather than just on dismiss? and is it possible to hide the `dismiss` message like in the material guide too?

Reply:
After thinking about this more and realizing that in material there can be more than one action, I replaced this with an `actions?: DNode | Dnode[]` property, allowing the user to control what actions are there, completely.

Examples:

```tsx
<Snackbar
	open={this._showSnackbar}
	message="Test Snackbar"
	actions={<Button onClick={() => this._showSnackbar = false}>Dismiss</Button>}
/>
```

and 

```tsx
<Snackbar
	open={this._showSnackbar}
	type="error"
	message="Task failed."
	actions={[
		<Button onClick={() => retry()}>Retry</Button>,
		<Button onClick={() => this._showSnackbar = false}>Dismiss</Button>
	]}
/>
```

Reply:
@matt-gadd Thinking about this more... should it maybe be a property like `actionsRenderer?: () => DNode | DNode[]`?